containers/daap-athena-load/src/var/task/athena_load_handler.py

```diff
@@ -28,29 +27,37 @@
 def handler(event, context):
     bucket_name = event["detail"]["bucket"]["name"]
     file_key = event["detail"]["object"]["key"]
-
     full_s3_path = os.path.join("s3://", bucket_name, file_key)
-    config = get_data_product_config(full_s3_path)
+
+    extraction = ExtractionConfig.parse_from_uri(full_s3_path)
+    data_product = extraction.data_product_config
 
     logger.add_extras(
         {
             "lambda_name": context.function_name,
-            "data_product_name": config["database_name"],
-            "table_name": config["table_name"],
+            "data_product_name": data_product.curated_data_table.database,
+            "table_name": data_product.curated_data_table.name,
         }
     )
-    logger.info(f"config: {config}")
     logger.info(f"file is: {full_s3_path}")
+    logger.info(
+        f"config: {extraction.timestamp=} {data_product.raw_data_table=} {data_product.curated_data_table=}"
+    )
+
     metadata_types, metadata_str = infer_glue_schema(
         full_s3_path, "data_products_raw", logger=logger
     )
 
+    glue_client = boto3.client("glue")
+    athena_client = boto3.client("athena")
+    s3_client = boto3.client("s3")
```

Comment:
I think a few of these are defined at the beginning of the file

Reply:
Yeah, they used to be, but I moved them into the handler so that their instantiation during the lifecycle of the tests (otherwise some stubbing broke).

I've since discovered a few different ways of handling import issues here: http://docs.getmoto.org/en/latest/docs/getting_started.html#what-about-those-pesky-imports

So if you prefer I can move this back and try and workaround this another way. Not sure if there is any other advantage/disadvantage to instantiating at module level vs handler level.

Reply:
leave it like this by all means, but remove the doubles from the beginning of the file eg athena and s3 on L10/11?

Reply:
whoops - I didn't spot those duplicates. I've changed this now so they're defined at the module level but can be overridden by tests.